templates/referent/_layout.html.twig

```diff
@@ -111,8 +111,8 @@
                             {% endif %}
 
                             {% if isReferent %}
-                                <li class="{{ is_active_route(app.request, 'app_territorial_council_referent_*') ? 'active' }}">
-                                    <a href="{{ path('app_territorial_council_referent_designations_list') }}">Instances</a>
+                                <li class="{{ is_active_route(app.request, 'app_territorial_council_referent_*') or is_active_route(app.request, 'app_referent_territorial_council_*') ? 'active' }}">
```

Comment:
maybe we can rename your route prefix
to not have this 2 prefixes:
```
'app_territorial_council_referent_*'
'app_referent_territorial_council_*'
```

Reply:
I need distinguish 2 spaces here : https://github.com/EnMarche/en-marche.fr/pull/4749/files#diff-eeeb9ccf7e8b0f98d1b9a2f967c75e5bR3
It's another submenu.
<img width="1238" alt="Capture d’écran 2020-08-31 à 16 59 19" src="https://user-images.githubusercontent.com/15227655/91734583-64206100-ebab-11ea-892c-43135085837d.png">